packages/pluggableWidgets/signature-native/src/Signature.xml

```diff
@@ -0,0 +1,55 @@
+    </icon>
+    <properties>
+        <propertyGroup caption="General">
+            <propertyGroup caption="General">
+                <property key="imageAttribute" type="attribute">
+                    <caption>Attribute</caption>
+                    <description/>
+                    <attributeTypes>
+                        <attributeType name="String"/>
+                    </attributeTypes>
+                </property>
+            </propertyGroup>
+            <propertyGroup caption="Buttons">
+                <property key="buttonCaptionClear" type="textTemplate" required="false">
+                    <caption>Button caption clear</caption>
+                    <description>The caption for the clear button.</description>
```

Comment:
IMO description is not needed here.

Reply:
i would review with Connor, as this is in the things we discussed last meeting about reviews and widget dev process.